### ISSUE TITLE
Split "True and false variants are identical" from "Nested ternary operator" inspection.

### ISSUE THREAD

**Issue by donquixote** [kalessil/phpinspectionsea#823]
I saw this inspection warning:

> Confusing constructs / Nested ternary operator / True and false variants are identical, most probably this is a bug.

Example code:

```php
function foo(bool $x) {
  return $x ? [] : [];
}
```

The code was underlined in red, even though the inspection is only configured as "warning".

I use code like this as temporary placeholder code. It does make sense to show a warning, but the red underlining is too much.

Problem is, there is only one item in the configuration, and it is for "Nested ternary operator". There is no option to configure "True and false variants are identical, most probably this is a bug." separately.

## Proposed change

Split this inspection out of "Nested ternary operator". It has nothing to do with it.

This allows to configure separate on/off and severity level for this inspection.

**Comment by kalessil** [comment]
I'll relocate the check to "Suspicious ternary operator" instead (respecting verbosity settings ofcourse).

**Comment by kalessil** [comment]
Done, relocated into "Suspicious ternary operator" inspection.
